Dockerfile_dependencies

```diff
@@ -1,16 +1,14 @@
-FROM alpine:3.20
+FROM ubuntu:22.04
 
 ARG TARGETPLATFORM
 
-RUN apk update && apk add --no-cache py3-pip \
-    build-base=0.5-r3 \
-    cmake=3.29.3-r0 \
-    bash=5.2.26-r0 \
-    linux-headers=6.6-r0 \
-    boost-build=1.84.0-r0 \
-    onetbb=2021.12.0-r0
```

Comment:
Why don't we need to install tbb here? I would expect that tbb needs to be installed so that the build later in `Dockerfile` succeeds? Is it maybe already installed?